src/ucp/core/ucp_mm.c

```diff
@@ -561,6 +629,9 @@ ucp_memh_register_internal(ucp_context_h context, ucp_mem_h memh,
 out_close_dmabuf_fd:
     UCS_STATIC_ASSERT(UCT_DMABUF_FD_INVALID == -1);
     ucs_close_fd(&reg_params.dmabuf_fd);
+    if (memh->flags & UCP_MEMH_FLAG_MLOCKED) {
+        munlock(address, length);
+    }
```

Comment:
why is munlock needed here?

Reply:
pls check if needed, probably we need that if there is no any md which registered gva